app/addons/auth/resources.js

```diff
@@ -136,7 +133,7 @@ Auth.Session = CouchdbSession.Session.extend({
     }
   },
 
-  createAdmin: function (username, password, login, node) {
+  createAdmin(username, password, login, node) {
```

Comment:
i would like to see those style changes in a separate commit, so the diff for the actual change is easier to reason about